test/Tool/range3.opt

```diff
@@ -0,0 +1,12 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: %FileCheck %s < %t
+
+; CHECK: LGTM
+%0:i32 = var (range=[2, 5))
+%1:i32 = var (range=[3, 5))
```

Comment:
%1 doesn't seem to be used anywhere?

Reply:
Thanks for catching this test, I had something else in my notepad and wrote a different test here. I have fixed it.